packages/server/src/validation.ts

```diff
@@ -10,4 +10,4 @@ export const accountSchema = Joi.object({
     })
     .required(),
   password: Joi.string().alphanum().min(6).max(30).required(),
-});
+}).required();
```

Comment:
What does this required() do ? :))

Reply:
An empty body would pass this validation if we don't require it!

Reply:
Ooh I see good one ! 👍